rdmd.d

```diff
@@ -571,12 +591,12 @@ private int exec(string[] args)
     return run(args, null, true);
 }
 
-// Given module rootModule, returns a mapping of all dependees .d
-// source filenames to their corresponding .o files sitting in
+// Given module rootModule, compiles it to rdmd.root.o and returns a mapping of
+// all dependees .d source filenames to their corresponding .o files sitting in
 // directory workDir. The mapping is obtained by running dmd -v against
 // rootModule.
 
-private string[string] getDependencies(string rootModule, string workDir,
+private string[string] compileRootAndGetDeps(string rootModule, string workDir,
         string objDir, string[] compilerFlags)
 {
     immutable depsFilename = buildPath(workDir, "rdmd.deps");
```

Comment:
A bit unrelated, but this could also be the cause of the slight runtime difference. At line 640 we can use a `ctRegex`